engines/cryomni3d/versailles/toolbar.cpp

```diff
@@ -121,6 +121,8 @@ void Toolbar::updateZones() {
 		_inventoryOffset = 0;
 		_zones[10].secondary = true;
 		_zones[11].secondary = true;
+		inventoryIt = _inventory->begin();
+		inventorySelectedIt = _inventory->begin();
```

Comment:
For these two lines, I would use _inventory->end() to ensure we point on an invalid data.
That would be a logic error to use these variables in this state